Modules/System/WebDAV/src/Model/WebDAVContent.Table.al

```diff
@@ -0,0 +1,66 @@
+table 5678 "WebDAV Content"
+{
+    Access = Public;
+    DataClassification = SystemMetadata; // Data classification is SystemMetadata as the table is temporary
+    Caption = 'WebDAV Folder';
+    TableType = Temporary;
+    Extensible = false;
+
+    fields
+    {
+        field(1; "Entry No."; Integer)
+        {
+            Caption = 'Entry No.';
+            AutoIncrement = true;
+        }
+        field(10; Name; Text[250])
+        {
+            Caption = 'Title';
+        }
+        field(11; "Full Url"; Text[2048])
+        {
+            Caption = 'Full Url';
+        }
+        field(12; "Relative Url"; Text[2048])
+        {
+            Caption = 'Relative Url';
+        }
+        field(13; Level; Integer)
+        {
+            Caption = 'Level';
+        }
+        field(20; "Is Collection"; Boolean)
+        {
+            Caption = 'Is Collection';
+        }
+        field(22; "Content Type"; Text[2048])
+        {
+            Caption = 'Content Type';
+        }
+        field(23; "Content Length"; Integer)
+        {
+            Caption = 'Content Length';
+        }
+        field(30; "Creation Date"; DateTime)
```

Comment:
I often add these fields too, and there might be a reason, but isn't this handled by the system fields? (SystemCreatedAt, SystemModifiedAt)

Reply:
it's not the creation date of the record but of the file on the webdav file system

Reply:
Understood.